modules/go-setting-environment-annotations.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-setting-environment-annotations_{context}']             
+= Setting up environment labels and annnotations
+
+[discrete]
+== Environment labels:
+
+The environment application manifest should include `openshift.gitops/environment` label, and `destination.namespace` set. The set value for both the environment name and destination namespace should be the same.
```

Comment:
```suggestion
The environment application manifest must include `openshift.gitops/environment` label, and `destination.namespace` set. The set value for both the environment name and destination namespace must be identical.
```
- Per the ISG: "Do not use _should_ to refer to an action that must be performed." 

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s